src/components/blogContent/BlogElement.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { makeStyles, TextField, Typography } from "@material-ui/core";
+type BlogElementProps = {
+  fieldName: string;
+  multiline: boolean;
+  minRows: number;
+};
+const useStyles = makeStyles((theme) => ({
+  fieldContainer: {
+    padding: theme.spacing(3),
+  },
+}));
+const BlogElement = (props: BlogElementProps) => {
```

Comment:
Updated but how about just using destructuring props? It will be less code. What do you think?

Reply:
It's an option you can use. I tend to like explicitly declared interfaces and type usage without the unpacking in case you change the interface later, but they're "functionally equivalent" so both work :)